rest/models/indicators.go

```diff
@@ -0,0 +1,419 @@
+package models
+
+// GetSMAParams is the set of parameters for the GetSMA method.
+type GetSMAParams struct {
+	// The ticker symbol of the stock/equity.
+	Ticker string `validate:"required" path:"ticker"`
+
+	// The size of the timespan multiplier for the underlying aggregates.
+	Multiplier *int `query:"multiplier"`
```

Comment:
did y'all remove this param from the API? not seeing it in the docs anymore

Reply:
ah nice catch